### PR TITLE
Feature/force update

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: Puppet/Puppet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -820,6 +820,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CFLAGS = "-fno-optimize-sibling-calls";
 				SDKROOT = iphoneos;
+				TARGETED_DEVICE_FAMILY = "1,2";
```
Comment (TroubleMakerBen) [inline Puppet/Puppet.xcodeproj/project.pbxproj]: why the new entry?

Comment (jaelim-ms) [inline Puppet/Puppet.xcodeproj/project.pbxproj]: That's correct. I enabled the applications for iPad.

File: MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m [hunk 2]
```diff
@@ -484,17 +487,20 @@ - (void)startDownload:(MSReleaseDetails *)details {
 #if TARGET_IPHONE_SIMULATOR
   MSLogWarning([MSDistribute logTag], @"Couldn't download a new release on simulator.");
 #else
+
+  /*
+   * We've seen the behavior on iOS 8.x devices in HockeyApp that it doesn't download until the application
+   * goes in background by pressing home button. Simply exit the app to start the update process.
+   * For iOS version >= 9.0, we still need to exit the app if it is a mandatory update.
+   */
+  BOOL exitBeforeDownload =
```
Comment (jaelim-ms) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m]: I got your point.

**Comment by jaelim-ms** [comment]
@clpolet That can be a separate task. Android displays toast message for the case, we need to do a similar way to inform to end users.

**Pull request merged**